lldb/source/Plugins/Protocol/MCP/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_lldb_library(lldbPluginProtocolServerMCP PLUGIN
```

Comment:
Should this be behind a build flag?

I could see some distributions disabling this feature for security reasons.

Once an MCP server is running, it can run arbitrary commands in the debugger and may even be able to spawn arbitrary processes.